routes/v1/post-tree.js

```diff
@@ -33,28 +37,23 @@ let endpoint = (req, res, next) => {
   const user_ip = userHash(req)
 
   client.connect()
-  const query = [
-    "UPDATE trees",
-    "  SET type = $1, description = $2, img = $3, added_by = $4, added_at = now()",
-    "  WHERE ssm_key = $5",
-  ].join(" ")
-  client.query(
-    query,
-    [type, sanitizeText(desc), img, user_ip, key],
-    (err, response) => {
-      client.end()
-      if (err) {
-        return next(
-          new InternalServerError(`Error connecting to database: ${err}`)
-        )
-      }
-      if (response.rowCount === 0) {
-        return next(
-          new InternalServerError(`No tree found to update for key ${key}`)
-        )
-      }
-      res.json({})
+  const query = ["UPDATE trees SET added_by = $2, added_at = now()"]
+  query.push(" , type = $3")
+  query.push(" , description = $4")
+  query.push(" , img = $5")
+  query.push(" WHERE ssm_key = $1")
+  var values = [key, user_ip, type, sanitizeText(desc), img]
```

Comment:
I'd suggest sticking to const/let (I would have thought the linter would have warned agains this?)